_resources/port1.0/group/haskell_stack-1.0.tcl

```diff
@@ -57,31 +57,41 @@ post-extract {
 compiler.library_path
 compiler.cpath
 
-default configure.cmd       {${prefix}/bin/stack}
+options haskell_stack.bin haskell_stack.default_args
+default haskell_stack.bin   ${prefix}/bin/stack
+default haskell_stack.default_args \
+    {--stack-root [option haskell_stack.stack_root] \
+         --with-gcc ${configure.cc} \
+         --allow-different-user \
+         [haskell_stack.system_ghc_flags]}
+
+options haskell_stack.use_init
+default haskell_stack.use_init yes
+
+pre-configure {
+    if {${haskell_stack.use_init} eq yes} {
+        if {[glob -nocomplain ${worksrcpath}/stack*.yaml] == ""} {
```

Comment:
AFAIK, `stack` will only use `stack.yaml` unless you explicitly provide a `--stack-yaml` command line option. Therefore, we should be searching only for `stack.yaml`.

@amake Would you please sanity check this assertion? If I’m wrong, it means there needs to be an option about which `stack*.yaml` to use.

Reply:
You're right, it only looks at `stack.yaml`. I had assumed something different based on a hunch, but that was wrong.

I've added a `haskell_stack.yaml` option to allow specifying the file used.